src/frontend/admin/src/components/Title.js

```diff
@@ -118,7 +118,17 @@ function TitleTableRow({ data, onShow, onEdit, onDelete }) {
 
     return (<TableRow key={data.id} sx={{ backgroundColor: (data.shown ? activeRowColor : undefined) }}>
         <TableCell component="th" scope="row" align={"left"} key="__show_btn_row__">
-            <ShowPresetButton onClick={onShow} active={data.shown}/>
+            <Box display="flex">
+                <ShowPresetButton className='block' onClick={onShow} active={data.shown}/>
+                <IconButton color="primary" onClick={() => {
+                    data.settings.preset = "title_left.svg";
```

Comment:
Hardcoded value. This preset needs to be able to change. It's good for what it's worth. But maybe we can add some sort of logic or interface to be able to add more such buttons, or less of such buttons?